sipi/scripts/convert_from_path.lua

```diff
@@ -124,6 +124,20 @@ if media_type == IMAGE then
         return
     end
 
+    local check
+    success, check = fullImg:mimetype_consistency(originalMimeType, originalFilename)
```

Comment:
maybe you need to add 

```
ocal submitted_mimetype
success, submitted_mimetype = server.parse_mimetype(originalMimeType)

local check
success, check = fullImg:mimetype_consistency(submitted_mimetype.mimetype, originalFilename)

```

Reply:
I've added that in a1a37cc, but Sipi returns the same HTTP 500 error. I actually think that since this is an API v1 or Sipi bug, and this PR only adds functionality to API v2, there's no reason to hold back this PR because of it.

Reply:
Ok , and you are going to change the v1 behaviour anyway. If your fix doesn't solve the problem, could you open an issue in Sipi?

Reply:
In other words, this problem was there before, and this PR doesn't affect it.

Reply:
I understand, but the code shouldn't have been just commented out in https://github.com/dasch-swiss/knora-api/pull/1459/files#diff-4a1a2aedd2d8b7dee2f824e33b243ca1R123

Reply:
It's ok for me to make separate issues (Knora and/or Sipi) and move on with this PR

Reply:
Perhaps, but that's a different PR. You can't expect every PR to fix every bug that already existed before the PR. You're reviewing **this** PR.

Reply:
#1531

Reply:
#1531

Reply:
> Perhaps, but that's a different PR. You can't expect every PR to fix every bug that already existed before the PR. You're reviewing this PR.

I agree. But this is kind of a critical issue (Sipi is expected to recreate the image file in its original format), this is why I kept insisting. If it had been a detail, it would not have mattered that much.